From 923ebd634949c11444fbdb6b1ee9ac62ac538ec7 Mon Sep 17 00:00:00 2001
From: Heiko Becker <mail@heiko-becker.de>
Date: Thu, 28 Nov 2024 21:43:15 +0100
Subject: [PATCH] src/ipc.h: Include <stdint.h> for uint32_t

Without this the build fails with the upcoming gcc 15:

In file included from ../wpebackend-fdo-1.14.2/src/ipc.cpp:26:
../wpebackend-fdo-1.14.2/src/ipc.h:36:36: error: 'uint32_t' has not been declared
   36 |     virtual void didReceiveMessage(uint32_t messageId, uint32_t messageBody) = 0
---
 src/ipc.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/ipc.h b/src/ipc.h
index c274529..f9945eb 100644
--- a/src/ipc.h
+++ b/src/ipc.h
@@ -26,6 +26,7 @@
 #pragma once
 
 #include <gio/gio.h>
+#include <stdint.h>
 #include <memory>
 
 namespace FdoIPC {
-- 
2.48.1

